src/menus/block.ts

```diff
@@ -107,6 +107,12 @@ export default function blockMenuItems(
       icon: ImageIcon,
       keywords: "picture photo",
     },
+    {
+      name: "file",
+      title: "File",
+      icon: ImageIcon,
```

Comment:
Going to need a unique icon here – there isn't a simple "file" icon in the set, but I can get one added.

Reply:
Yes please get it added will use the same Icon here